source/Ship.cpp

```diff
@@ -423,6 +423,8 @@ void Ship::Load(const DataNode &node)
 			cargo.Load(child);
 		else if(key == "crew" && child.Size() >= 2)
 			crew = static_cast<int>(child.Value(1));
+		else if(key == "crew equivalent" && child.Size() >= 2)
+			crewEquivalent = static_cast<int>(child.Value(1));
```

Comment:
`"crew equivalent"` is never set by this implementation.
All the `else if`s from "crew" to "hull" are the ship's current levels, *not* the ship's attributes. Attributes are already handled up on line 193.
```suggestion
```